kerncraft/iaca_marker.py

```diff
@@ -13,6 +13,8 @@
 import re
 from six.moves import map
 from six.moves import input
+import subprocess
+from distutils.spawn import find_executable
```

Comment:
Since you seem to be a fan of PEP8 and PEP257 :)

> Imports should be grouped in the following order:
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
>
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports